R/dataframe__frame.R

```diff
@@ -60,6 +60,23 @@
 #'
 #' `$width` returns the number of columns in the DataFrame.
 #'
+#' @section Conversion to R data types considerations:
+#' Polars objects, such as [the DataFrame class object][DataFrame_class]
+#' can be converted to R objects via some way. For example, using the
+#' [`as.data.frame()`][S3_as.data.frame] generic function.
+#' When converting Polars objects to R objects, each type in the Polars object is
+#' converted to an R type.
+#' In some cases, an error may occur because the conversion is not appropriate.
+#' In particular, there is a high possibility of an error when converting
+#' a [Datetime][DataType_Datetime] type without a time zone.
+#' A [Datetime][DataType_Datetime] type without a time zone in Polars is converted
+#' to the [POSIXct] type in R, with taking into account the time zone in which
+#' the R session is running (which can be checked with the [Sys.timezone()]
+#' function). In this case, if ambiguous times are included, a conversion error
```

Comment:
It would be good to give an example of ambiguous time

Reply:
Done in 1ff2424